src/Esfa.Vacancy.Register.Application/Queries/SearchApprenticeshipVacancies/SearchApprenticeshipVacanciesQueryHandler.cs

```diff
@@ -1,29 +1,24 @@
-﻿using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
+﻿using System.Threading.Tasks;
 using Esfa.Vacancy.Register.Application.Interfaces;
-using Esfa.Vacancy.Register.Domain.Entities;
-using Esfa.Vacancy.Register.Domain.Repositories;
 using FluentValidation;
-using FluentValidation.Results;
 using MediatR;
 
 namespace Esfa.Vacancy.Register.Application.Queries.SearchApprenticeshipVacancies
 {
     public class SearchApprenticeshipVacanciesQueryHandler : IAsyncRequestHandler<SearchApprenticeshipVacanciesRequest, SearchApprenticeshipVacanciesResponse>
```

Comment:
seeing as we are renaming stuff, I'd suggest that this is `ApprenticeshipsSearchQueryHandler.cs`, likewise for other request, response objects, validators etc.

Reply:
I agree with Scott although using the singular ApprentishipSearch... sits better with me.